cmd/monaco/generate/deletefile/deletefile.go

```diff
@@ -144,9 +145,16 @@ func generateDeleteFileContent(environment string, projects []project.Project, a
 		for _, cfgs := range cfgsPerType {
 			for _, c := range cfgs {
 				if apis.Contains(c.Coordinate.Type) {
-					val, err := c.Parameters[config.NameParameter].ResolveValue(parameter.ResolveContext{ParameterName: config.NameParameter})
+					nameParam := c.Parameters[config.NameParameter]
+
+					if nameParam.GetType() == reference.ReferenceParameterType {
```

Comment:
pls add a short comment why we're leaving out that case?

Reply:
done.
On second thought I think we could actually improve the generator to resolve references that are not the IDs of other configs.
We'd just need to sort and build an entity map for each config. If I find the time I'll address this in a followup PR

Reply:
cool, or create a ticket otherwise :)